src/styles/blocks/page.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+  transition: all 0.3s;
```

Comment:
While in this task, the solution can be acceptable (because it says "make all changes smooth"), overall, it's very inefficient, and I highly recommend avoiding the use of the `*` selector along with `transition` propery.